.cirrus.yml

```diff
@@ -23,6 +23,7 @@ timeout_in: 60m
 task:
     name: "Validate"
     alias: "validate"
+    skip: "changesInclude('**.md', 'LICENSE')"
```

Comment:
This raises huge warning flags. As I read this, it means "skip if any .md or LICENSE file was touched". That is not the same as "skip if **ONLY** .md or LICENSE files were touched". But I cannot make any sense of the [cirrus docs](https://cirrus-ci.org/guide/writing-tasks/#supported-functions) nor their example, nor can I find any useful examples on the net, and I don't really want to dive into the Cirrus source code right now.

Reply:
oops.  it should be `!changesInclude('**md', 'LICENSE')`

Reply:
Hmmm, nope, inverting the condition doesn't work as the docs suggest :confused:

Okay I have an idea...